.circleci/config.yml

```diff
@@ -70,11 +70,6 @@ workflows:
     when: << pipeline.parameters.run-integration-tests >>
     jobs:
       - integration-test_saucectl
-      - saucectl/saucectl-run:
```

Comment:
You don't need to define this in the yaml anymore?

Reply:
This removed element is the `job` (which is composed of `commands`).
Part of that rewrite is to only rely on the command, as there is no benefits of providing a job definition.